DMTN-222.tex

```diff
@@ -313,23 +323,64 @@ \subsection{Distribution}
   \hline
 \end{tabular}
 
-\subsubsection{Summit Cleanup}
+\subsubsection{Distribution With Rucio}
+
+The way to copy calibration files from \texttt{\/repo\/main} to the Summit, BTS, and the other DFs is via \texttt{rucio}.
```

Comment:
"The preferred way"?

Reply:
This is the only way it can be put into the calib bucket at Summit and BTS without super-admin-special database hacking.